src/main/java/fr/uge/chargepointconfiguration/chargepoint/ocpp/ocpp2/data/Component.java

```diff
@@ -0,0 +1,9 @@
+package fr.uge.chargepointconfiguration.chargepoint.ocpp.ocpp2.data;
+
+/**
+ * Defines a component of the machine.
```

Comment:
what is a component of the machine ?

Reply:
I don't really know, the OCPP 2.0 documentation is not really clear on this type. Will search

Reply:
I deleted it